scripts/npmPublishBeta.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+echo "This script will bump npm version and push changes to new branch"
+
+# echo "checkout new branch for version bump"
+git checkout -b npmRelease
+
+## configure npm to sign commit
+npm config set sign-git-tag true
+
+version=$(npm version prerelease -m "Bumped to version %s")
+
+git tag -l "v1*"
```

Comment:
was this for visibility/debugging or does this so something else?

Reply:
for visibility when running the script.  I'll change it to "echo $version" instead.